Quaver.Shared/Screens/Edit/Plugins/EditorPluginUtils.cs

```diff
@@ -125,71 +125,68 @@ public static EditorLayerInfo CreateEditorLayer(string name, bool hidden = false
         /// <summary>
         /// </summary>
         /// <param name="type"></param>
-        /// <param name="arg1"></param>
-        /// <param name="arg2"></param>
-        /// <param name="arg3"></param>
-        /// <param name="arg4"></param>
+        /// <param name="args"></param>
         /// <returns></returns>
-        public static IEditorAction CreateEditorAction(EditorActionType type, DynValue arg1 = null, DynValue arg2 = null, DynValue arg3 = null, DynValue arg4 = null)
+        public static IEditorAction CreateEditorAction(EditorActionType type, params DynValue[] args)
```

Comment:
Is there any way to avoid having this method? (Like creating the objects directly in Lua after they're exposed)

Reply:
I tried finding another way to do this, but as far as I can tell this is pretty much the only way to do it